test/functional/test_framework/messages.py

```diff
@@ -1724,8 +1724,11 @@ class msg_tx:
     __slots__ = ("tx",)
     msgtype = b"tx"
 
-    def __init__(self, tx=CTransaction()):
-        self.tx = tx
+    def __init__(self, tx=None):
```

Comment:
missing changes for class CMerkleBlock (same bug):

```diff
--- a/test/functional/test_framework/messages.py
+++ b/test/functional/test_framework/messages.py
@@ -1069,9 +1069,16 @@ class CPartialMerkleTree:
 class CMerkleBlock:
     __slots__ = ("header", "txn")
 
-    def __init__(self, header=CBlockHeader(), txn=CPartialMerkleTree()):
+    def __init__(self, header=None, txn=None):
+        if header is None:
+            self.header = CBlockHeader()
+        else:
+            self.header = header
         self.header = header
-        self.txn = txn
+        if txn is None:
+            self.txn = CPartialMerkleTree()
+        else:
+            self.txn = txn
 ```

Reply:
fixed: https://github.com/dashpay/dash/pull/6373

Reply:
Thanks!